spec/controllers/posts_controller_spec.rb

```diff
@@ -2316,6 +2316,60 @@
       expect(assigns(:page_title)).to eq('Replies Owed')
     end
 
+    context "with views" do
+      render_views
+
+      def create_owed(user)
+        post = create(:post, user: user)
+        create(:reply, post: post)
+        post.mark_read(user)
+      end
+
+      it "succeeds" do
+        user = create(:user)
+        login_as(user)
+        create_owed(user)
+        get :owed
+        expect(response.status).to eq(200)
+        expect(response.body).to include('note_go_strong')
+      end
+
+      it "succeeds with dark" do
+        user = create(:user, layout: 'starrydark')
+        login_as(user)
+        create_owed(user)
+        get :owed
+        expect(response.status).to eq(200)
+        expect(response.body).to include('bullet_go_strong')
+      end
+    end
+
+    context "with hidden" do
+      let(:user) { create(:user) }
+
+      before(:each) do
+        login_as(user)
+
+        @unhidden_post = create(:post, user: user)
```

Comment:
Couldn't we use `let`s for these, instead of instance variables? I think that's more RSpec-y, and it should load them just fine so long as we still reference them in this `before(:each)`.